modules/vector-index/pages/vector-index-best-practices.adoc

```diff
@@ -0,0 +1,262 @@
+= Hyperscale and Composite Vector Index Best Practices
+:page-topic-type: guide
+:page-ui-name: {ui-name}
+:page-product-name: {product-name}
+:description: When creating and querying Hyperscale and Composite Vector indexes, you have several options to set that can affect the speed and accuracy of your results.
+:stem: 
+:page-toclevels: 3
+
+[abstract]
+{description}
+Couchbase has performed testing to determine how index creation and query options affect the performance of Hyperscale and Composite Vector indexes.
+The following sections describe the results of the testing and recommendations based on these results. 
+
+NOTE: Most of the tests were performed using the Hyperscale Vector index.
+However, many of the results also apply to Composite Vector indexes where the two index types share the same settings.
+
+== Tune Index Creation
+
+When you create a Hyperscale Vector index, you can set the parameters in the following list. 
+Each has the potential to affect the performance of creating the index and the speed and accuracy of queries that use the index.
+
+[#nlist]
+=== nList
+
+The `nList` setting controls the number of clusters (centroids) in the index. 
+You set the `nList` by supplying a value after the `IVF` in the `description` parameter when you create your index.
+This value affects the accuracy of the index and the speed of queries that use it.
+The default value for this value depends on the size of your dataset.
+For datasets with less than 1 million vectors, the default value is the number of vectors divided by 1000.
+For datasets with more than 1 million vectors, the default value is the square root of the number of vectors.
+
+
+Tests show that increasing `nList` beyond the default does not affect the build time of the index. 
+However, it does improve the queries per second (QPS) that the index can handle.
+It also lowers the latency of queries that use the index.
+
+
+*Recommendation:* if you find that your query throughput and latency is not meeting your needs, you can try rebuilding the index with a larger `nList` value.
+
+=== TrainList
+
+The `TrainList` setting controls the number of vectors that Couchbase {product-name} considers when searching for centroids in the dataset.
+You set it using the `trainlist` key in the `WITH` clause when you create your index.
+The default value is approximately 50 times the number of centroids in the index.
```

Comment:
This is old information.  Currently for trainlist - Default - If items_count < 10000, sample everything. Otherwise, higher of 10% of items_count or 10x the number of centroids, with a max of 1M.

https://docs.google.com/document/d/1c499ChQD8CEU3Lh6UeMHHstgxIgJZGSV6tyGQRjnL9Q/edit?tab=t.0#heading=h.wlm05svaxxtk